pops/models.py

```diff
@@ -262,6 +262,26 @@ class PestInformation(models.Model):
     date_created = models.DateTimeField(verbose_name = _("date created"), auto_now = False, auto_now_add = True)
     date_updated = models.DateTimeField(verbose_name = _("date updated"), auto_now = True, auto_now_add = False)
     staff_approved = models.BooleanField(verbose_name = _("approved by staff"), help_text="Sample help text.", default = False)
+    invasive = models.BooleanField(verbose_name = _("invasive"), help_text="Is the organism invasive in the US?", default = True)
+    HOST_CHOICES = (
+        ("ANIMAL", "Animal"),
+        ("PLANT", "Plant"),
+    )
+    host_type = models.CharField(verbose_name = _("host type"), help_text="Choose what system type this pest/pathogen infects.", max_length = 30,
+                    choices = HOST_CHOICES,
+                    default = "PLANT", blank = False)     
+    ORGANISM_CHOICES = (
+        ("PEST", "Pest (e.g. insect)"),
+        ("PATHOGEN", "Pathogen (e.g. disease)"),
+    )
+    organism_type = models.CharField(verbose_name = _("organism type"), help_text="Choose whether this is a pest or pathogen.", max_length = 30,
+                    choices = ORGANISM_CHOICES,
+                    default = "PEST", blank = False) 
+    arrival_year = models.PositiveSmallIntegerField(verbose_name = _("first year found in US"), help_text="The first year that it was identified in the US.", blank=True, null=True, default = None, validators = [MinValueValidator(1700), MaxValueValidator(2025)])
```

Comment:
Change max to 2100 so we don't end up forgetting this in 6 for a pest discovered in 2026.

Reply:
Done